### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by TheLoneWolfling** [incrediblesound/huo#48]
We should have a version number.

Personally, I'm a fan of semantic versioning, but it's ultimately your call.

(" three-part version number: major version; minor version; and patch. The patch number is incremented for minor changes and bug fixes which do not change the software's application programming interface (API). The minor version is incremented for releases which add new, but backward-compatible, API features, and the major version is incremented for API changes which are not backward-compatible")

**Comment by incrediblesound** [comment]
Yes, I am a fan of that system as well. We could work on dev_x.x branches and then merge them into master or a branch called prod_x.x or something like that.

**Comment by incrediblesound** [comment]
By the way did you get the org invite?

**Comment by TheLoneWolfling** [comment]
Yes I did.

Shall we call the current state of Huo 1.0.0, then? It seems "stable" enough for now.
